drivers/aks/aks_driver.go

```diff
@@ -1,75 +1,140 @@
 package aks
 
 import (
-	"regexp"
-	"strings"
-
 	"context"
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"time"
-
-	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice"
-	"github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2017-05-10/resources"
+	containerservice_20180331 "github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2018-03-31/containerservice"
+	operationalinsights_20151101preview "github.com/Azure/azure-sdk-for-go/services/preview/operationalinsights/mgmt/2015-11-01-preview/operationalinsights"
+	resources_20170510 "github.com/Azure/azure-sdk-for-go/services/resources/mgmt/2017-05-10/resources"
```

Comment:
If we're not pulling in multiple versions of these imports, can we leave them with the default imported name? This may be confusing down the line if we user the custom name `resources_20170510` but someone updates it to a later version and forgets to update the name.

Reply:
Fixed